src/main/java/net/fabricmc/installer/Handler.java

```diff
@@ -16,11 +16,10 @@
 
 package net.fabricmc.installer;
 
-import java.awt.Color;
-import java.awt.Container;
-import java.awt.FlowLayout;
-import java.io.IOException;
-import java.util.function.Consumer;
+import net.fabricmc.installer.util.ArgumentParser;
```

Comment:
I don't see why these imports need to be moving at all

Reply:
Woops, blame Intellij for that.